ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml

```diff
@@ -336,6 +336,82 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )edge-ci-index,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
With the existing configuration it will always run, and not block merging the PR
So if you intended it to only be triggered manually, you should change this to false

Reply:
Oh gotcha, thanks for letting me know! Will change this and add `run_if_changed` too